src/models/event.ts

```diff
@@ -63,6 +63,7 @@ export interface IUnsigned {
     "transaction_id"?: string;
     "invite_room_state"?: StrippedState[];
     "m.relations"?: Record<RelationType | string, any>; // No common pattern for aggregated relations
+    "io.element.relation_thread_id"?: string;
```

Comment:
I believe we should have the parent `relates_to` field copied into the children `unsigned`. That makes it less thread related, and better to write an MSC for IMO

```suggestion
    "io.element.parent.relates_to"?: {
    	rel_type: RelationType;
    	event_id: string;
    };
```

Reply:
I'm not suggesting this be the shape the MSC uses, the MSC should probably follow https://spec.matrix.org/v1.7/client-server-api/#aggregations-of-child-events - this is just a workaround for the problem at hand

Reply:
Updated to match https://github.com/matrix-org/matrix-spec-proposals/pull/4023